server/selinux/server/relabel.sh

```diff
@@ -21,6 +20,7 @@ fi
 # If upgrading from before 2.7.0
 if [[ $1 < '2.7.0' ]]
 then
+    /sbin/restorecon -i -R /usr/share/pulp/srv
```

Comment:
This should get moved to its own if block which is guarded at '2.8.0'. Otherwise it won't run when users upgrade from 2.7.0 -> 2.8.0. This is required for correctness